connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSinkTaskThreadedTest.java

```diff
@@ -147,7 +150,6 @@ public void testPollsInBackground() throws Exception {
             workerTask.iteration();
         }
         workerTask.stop();
-        workerTask.awaitStop(Long.MAX_VALUE);
```

Comment:
In particular, it seems like this and `WorkerSinkTaskTest` aren't too different, but one works with `poll()`, the other `iteration()`.